### PR TITLE
Update documents.

### PR DESCRIPTION
**Pull request by Random-Liu**
Fixes https://github.com/containerd/cri/issues/551.
Fixes https://github.com/containerd/cri/issues/619.
Fixes https://github.com/containerd/cri/issues/660.

Update documents.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ containerd. This github branch is for the `cri` plugin. See
 [standalone-cri-containerd branch](https://github.com/containerd/cri/tree/standalone-cri-containerd)
 for information about the standalone version of cri-containerd.*
 
+*Note: You need to [drain your node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) before upgrade from standalone `cri-containerd` to containerd with `cri` plugin.*
```
Comment (yujuhong) [inline README.md]: s/upgrade/upgrading

File: README.md [hunk 2]
```diff
@@ -8,7 +8,9 @@
 transitioning from a standalone binary that talks to containerd to a plugin within
 containerd. This github branch is for the `cri` plugin. See
 [standalone-cri-containerd branch](https://github.com/containerd/cri/tree/standalone-cri-containerd)
-for information about the standalone version of cri-containerd.*
+for information about the standalone version of `cri-containerd`.*
+
+*Note: You need to [drain your node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) before upgrade from standalone `cri-containerd` to containerd with `cri` plugin.*
```
Comment (yujuhong) [inline README.md]: s/upgrade/upgrading

Comment (Random-Liu) [reply README.md]: Done

**Comment by Random-Liu** [comment]
Apply LGTM based on https://github.com/containerd/cri/pull/696#pullrequestreview-106642069

**Pull request merged**